qiskit-superstaq/requirements.txt

```diff
@@ -1,5 +1,5 @@
 general-superstaq~=0.5.6
-matplotlib~=3.0
```

Comment:
Why do we even have matplotlib in qiskit-superstaq? Do/should we be making graphs?

Reply:
good question. i'll try removing it